packages/portal-proto/src/features/cDave/CDaveCard/CDaveCard.unit.test.tsx

```diff
@@ -37,7 +37,9 @@ describe("CDaveCard", () => {
     );
 
     expect(
-      getByRole("row", { name: "Select male 1,000 (66.67%)" }),
+      getByRole("row", {
+        name: "Select all the rows of Gender table male 1,000 (66.67%)",
```

Comment:
I don't think these changes to the tests are quite right, these rows shouldn't have this "Select all text". Instead of updating the tests, [these aria-labelledby](https://github.com/NCI-GDC/gdc-frontend-framework/blob/79bfcaedd84949670772f75f466ec7b76db6e8f0/packages/portal-proto/src/features/cDave/CDaveCard/CDaveTable.tsx#L127) should be updated probably to `aria-label={`Select ${displayName}`. 

(this is a good illustration of why using getByRole instead of getByTestId can be beneficial. I probably wouldn't have noticed the incorrect aria-labels on this component without the test).

Reply:
This has been fixed. Thank you for the help.